### ISSUE TITLE
[Bug] sd web ui 1.60版本 prompt highlighting 语法高亮 功能一直转圈圈 未能生效 

### ISSUE THREAD

**Issue by bbggkk** [lobehub/sd-webui-lobe-theme#366]
### 💻 系统环境 | Operating System

Windows

### 🌐 浏览器 | Browser

Chrome

### 🐛 问题描述 | Bug Description

等大佬更新么？

**Comment by etrosh** [comment]
Cannot translate properly to english, same issue on Opera/macOS. **Highlighting...** label is always visible after updating to 1.60 with a loading icon. The prompt doesn't update and the label cover portion of the text box.

**Comment by bbggkk** [comment]
> Cannot translate properly to english, same issue on Opera/macOS. **Highlighting...** label is always visible after updating to 1.60 with a loading icon. The prompt doesn't update and the label cover portion of the text box.

Highlighting Box is freezing, Yes,your understanding is correct.

**Comment by primeinc** [comment]
```javascript
// ==UserScript==
// @name         A1111 WebUI fix lobe theme
// @namespace    http://tampermonkey.net/
// @version      0.3
// @description  fix Lobe theme broken webui v1.6.0RC
// @author       SLAPaper
// @match        http*://localhost:7860/*
// @icon         https://www.google.com/s2/favicons?sz=64&domain=undefined.localhost
// @grant        none
// ==/UserScript==

function waitForElm(selector) {
    return new Promise(resolve => {
        if (document.querySelector(selector)) {
            return resolve(document.querySelector(selector));
        }

        const observer = new MutationObserver(mutations => {
            if (document.querySelector(selector)) {
                observer.disconnect();
                resolve(document.querySelector(selector));
            }
        });

        observer.observe(document.body, {
            childList: true,
            subtree: true
        });
    });
}

function patchDOM() {
    waitForElm('textarea[data-testid="textbox"][class^="svelte-"]').then((elm) => {
        elm.style.color = 'white';
    });
    waitForElm('.acss-m1wjsb[data-code-type="highlighter"]').then((elm) => {
        elm.style.display = 'none';
    });
    console.log('patchedDOM');
}

function handleTabClick() {
    patchDOM();
}

(function() {
    'use strict';

    // Run initially and every 5 seconds
    setInterval(patchDOM, 5000);

    // Listen for clicks on the document body with capturing phase
    document.body.addEventListener('click', function(e) {
        if (e.target && e.target.id === 'rc-tabs-0-tab-tab_img2img') {
            handleTabClick();
            console.log('img2img clicked');
        }
    }, true);
})();
```
This makes it go away, credit still goes to @SLAPaper since I wouldn't have bothered otherwise.

This has an added benefit of fixing the annoying issue of the text getting jumbled over time making it impossible to edit without cutting and pasting the prompt in notepad, changing it and pasting it back.

**Comment by bbggkk** [comment]
> ```js
> // ==UserScript==
> // @name         A1111 WebUI fix lobe theme
> // @namespace    http://tampermonkey.net/
> // @version      0.3
> // @description  fix Lobe theme broken webui v1.6.0RC
> // @author       SLAPaper
> // @match        http*://localhost:7860/*
> // @icon         https://www.google.com/s2/favicons?sz=64&domain=undefined.localhost
> // @grant        none
> // ==/UserScript==
> 
> function waitForElm(selector) {
>     return new Promise(resolve => {
>         if (document.querySelector(selector)) {
>             return resolve(document.querySelector(selector));
>         }
> 
>         const observer = new MutationObserver(mutations => {
>             if (document.querySelector(selector)) {
>                 observer.disconnect();
>                 resolve(document.querySelector(selector));
>             }
>         });
> 
>         observer.observe(document.body, {
>             childList: true,
>             subtree: true
>         });
>     });
> }
> 
> function patchDOM() {
>     waitForElm('textarea[data-testid="textbox"][class^="svelte-"]').then((elm) => {
>         elm.style.color = 'white';
>     });
>     waitForElm('.acss-m1wjsb[data-code-type="highlighter"]').then((elm) => {
>         elm.style.display = 'none';
>     });
>     console.log('patchedDOM');
> }
> 
> function handleTabClick() {
>     patchDOM();
> }
> 
> (function() {
>     'use strict';
> 
>     // Run initially and every 5 seconds
>     setInterval(patchDOM, 5000);
> 
>     // Listen for clicks on the document body with capturing phase
>     document.body.addEventListener('click', function(e) {
>         if (e.target && e.target.id === 'rc-tabs-0-tab-tab_img2img') {
>             handleTabClick();
>             console.log('img2img clicked');
>         }
>     }, true);
> })();
> ```
> 
> This makes it go away, credit still goes to @SLAPaper since I wouldn't have bothered otherwise.
> 
> This has an added benefit of fixing the annoying issue of the text getting jumbled over time making it impossible to edit without cutting and pasting the prompt in notepad, changing it and pasting it back.

THX man.
